Configuration/PyReleaseValidation/python/relval_2017.py

```diff
@@ -15,7 +15,8 @@
 
 numWFStart=10000
 numWFSkip=200
-numWFIB = [10060.0,10039.0,10046.0] ##2017 WFs to run in IB (TenMuE_0_200, TTbar, MinBias)
+#2017 WFs to run in IB (TenMuE_0_200, TTbar, ZEE, MinBias, TTbar PU, ZEE PU)
+numWFIB = [10021.0,10024.0,10025.0,10026.0,10023.0,10224.0,10225.0] 
```

Comment:
the numbering here do not match those workflows for 2015. It is simply a new mechanism of "auto-numbering" according to the order in a list, a mechanism newly introduced by Gaelle for the upgrade relvals.